### ISSUE TITLE
NFC Support

### ISSUE THREAD

**Issue by magdesign** [biemster/pmos-openbox-conky#1]
Thank you for this!

Just enabled NFC support on a op6 enchilada thanks to your patch.

dmesg | grep nfc
shows:
nfc: nfc_init: NFC Core ver 0.1

Since i failed adding your patches to the [linux_NfcFactoryTestApp](https://github.com/NXPNFCLinux/linux_NfcFactoryTestApp) (due to not enough knowledge) nor building [linux_libnfc-nci ](https://github.com/NXPNFCLinux/linux_libnfc-nci), i am wondering if you figured out a good solution/tool to read/write NFC with this device?

and why is this not implemented/upstream in postmarketOS yet? any huge bugs?

**Comment by biemster** [comment]
There was an attempt to upstream something similar a while ago, but that did not make it into pmos but I forgot why actually. A quick search on the sdm845 gitlab should get you to that ticket if you are interested. For the rest I got very little feedback on this, so I figured it's not a very popular feature and stopped pushing.

I stopped working on this when I figured that I can do everything in userspace in python, without official support in the os. My plan is to pick this up again when I'm daily driving my 6T and then just make custom python scripts for my use cases.
